### DISCUSSION TITLE
https://sqlpey.com/python/solved-5-effective-methods-to-fix-pycharm-no-module-error-when-importing-custom-modules/

### DISCUSSION THREAD

**Discussion by giscus** [CodeWithAbkhan/discussions-repo#1] [Announcements]
# https://sqlpey.com/python/solved-5-effective-methods-to-fix-pycharm-no-module-error-when-importing-custom-modules/

   
  This comprehensive guide provides solutions to fix the common PyCharm error: 'No module named my_module' when importing a Python script. Explore practical examples, FAQs, and troubleshooting tips.
  
  
  

https://sqlpey.com/python/solved-5-effective-methods-to-fix-pycharm-no-module-error-when-importing-custom-modules/

**Comment by wahab-weeinvent** [comment]
this method saved is just..
Method 3: Define Your File Locations Explicitly
